### DISCUSSION TITLE
collapsing on desktop mode

### DISCUSSION THREAD

**Discussion by edikurniawan-dev** [rappasoft/laravel-livewire-tables#712] [Feature Requests]
i have data to show in collapsing but on desktop mode, can you make collapsing on desktop mode, please.
thankyou

**Comment by howdu** [comment]
@rappasoft this would be really useful if by default you could have the column collapsed. Happy to submit a PR for this. Let me know your thoughts.

**Reply by howdu** [reply]
Doesn't look to be in the docs yet but with new release you can

> Disable columns from column select by default
> 
> Previously, there was no way to have a column hidden from column select by default. You can now [specify](https://rappasoft.com/docs/laravel-livewire-tables/v2/columns/column-selection#content-deselected-by-default):
> 
> ```php
> Column::make('Address', 'address.address')
>     ->deselected(),
> ```
